pluginframework/tfschema/list_nested_attribute.go

```diff
@@ -0,0 +1,80 @@
+package tfschema
+
+import (
+	dataschema "github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+)
+
+// ListNestedAttributteBuilder represents a list of complex (non-primitive) types.
+type ListNestedAttributeBuilder struct {
+	NestedObject       NestedAttributeObject
+	Optional           bool
+	Required           bool
+	Sensitive          bool
+	Computed           bool
+	DeprecationMessage string
+	Validators         []validator.List
+}
+
+func (a ListNestedAttributeBuilder) BuildDataSourceAttribute() dataschema.Attribute {
+	return dataschema.ListNestedAttribute{NestedObject: a.NestedObject.BuildDataSourceAttribute(), Optional: a.Optional, Required: a.Required, Sensitive: a.Sensitive, DeprecationMessage: a.DeprecationMessage, Computed: a.Computed, Validators: a.Validators}
+}
+
+func (a ListNestedAttributeBuilder) BuildResourceAttribute() schema.Attribute {
+	return schema.ListNestedAttribute{NestedObject: a.NestedObject.BuildResourceAttribute(), Optional: a.Optional, Required: a.Required, Sensitive: a.Sensitive, DeprecationMessage: a.DeprecationMessage, Computed: a.Computed, Validators: a.Validators}
```

Comment:
Suggestion: could we put each field on its own line? Same elsewhere that we are creating new objects.

Reply:
Sounds good

Reply:
Sounds good, somehow I'm surprised make fmt doesn't do it automatically